daemon/server/router/container/backend.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"io"
 
+	containerpkg "github.com/docker/docker/daemon/container"
```

Comment:
Alternatively, we could move it to `api/types/backend` to indicate "internal" (but public); ideally, I'd still want to get rid of these though 😂

Reply:
Note to self; with those types moving, perhaps it's a solid idea to put it there? Will have to have another look 

- https://github.com/moby/moby/pull/50534